src/core/Makefile.am

```diff
@@ -35,12 +35,16 @@ otherinclude_HEADERS = xlio_extra.h
 
 install-exec-hook:
 	rm -f $(DESTDIR)$(libdir)/libxlio.la
-	rm -f $(DESTDIR)$(libdir)/libxlio.a
 	rm -f $(DESTDIR)$(bindir)/state_machine_test
 	rm -f $(DESTDIR)$(bindir)/vlogger_test
+if DISABLE_XLIO_STATIC_LIB
+	rm -f $(DESTDIR)$(libdir)/libxlio.a
+endif
```

Comment:
Once default `--enable-static` is used, this block should be removed, because user will control whether static library is built or not.

Reply:
None of the experiments I did with --enable-static would work. Please suggest a fix.

Reply:
Done